### PR TITLE
CI: bump dependencies

### PR DESCRIPTION
**Pull request by zhenyavinogradov**
## Description

Update dependencies used by CI, to keep them up to date.

## Related issue(s)

https://issues.serokell.io/issue/OPS-1269

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

**Comment by gromakovsky** [comment]
Doesn't it duplicate #106?

**Comment by zhenyavinogradov** [comment]
#106 is from our updater bot, I'm not really sure how to interact with it to add update-related fixes or to stop new updates from breaking what's currently in review, and it only works for github and flakes for now, so I'm just creating new branches for OPS-1269.

File: flake.nix [hunk 1]
```diff
@@ -3,29 +3,18 @@
...
-    };
+  nixConfig = {
+    flake-registry = "https://github.com/serokell/flake-registry/raw/master/flake-registry.json";
+  };
 
-    nix-npm-buildpackage = {
-      url = "github:serokell/nix-npm-buildpackage";
-      inputs.nixpkgs.follows = "nixpkgs";
+  inputs = {
+    nixpkgs.url = "github:serokell/nixpkgs/087bf5c511ab4cfc6541fa4bf9c030b713a51ce5";
```
Comment (balsoft) [inline flake.nix]: Why is this pinned this way?

Comment (zhenyavinogradov) [reply flake.nix]: I forgot to remove this, removed now

**Review by balsoft — APPROVED** [review]

**Pull request merged**